diff -Naur src_old/drivers/media/dvb/dvb-core/dmxdev.h src/drivers/media/dvb/dvb-core/dmxdev.h
--- src_old/drivers/media/dvb/dvb-core/dmxdev.h	2012-01-08 07:01:24.044352814 +0900
+++ src/drivers/media/dvb/dvb-core/dmxdev.h	2012-01-08 07:01:33.156352722 +0900
@@ -105,7 +105,7 @@
 	struct dmx_frontend *dvr_orig_fe;
 
 	struct dvb_ringbuffer dvr_buffer;
-#define DVR_BUFFER_SIZE (10*188*1024)
+#define DVR_BUFFER_SIZE (100*188*1024)
 
 	struct mutex mutex;
 	spinlock_t lock;
