--- PKGBUILD	2017-04-18 20:28:46.406869371 +0900
+++ PKGBUILD	2017-04-18 20:30:12.576872728 +0900
@@ -75,7 +75,13 @@
     --enable-libxvid \
     --enable-netcdf \
     --enable-shared \
-    --enable-version3
+    --enable-version3 \
+    --enable-nonfree \
+    --enable-cuda \
+    --enable-cuvid \
+    --enable-libnpp \
+    --extra-cflags="-march=native -I/opt/cuda/include" \
+    --extra-ldflags="-L/opt/cuda/lib64"
 
   make
   make tools/qt-faststart
